monitoring/prometheus/alerts/ceph_default_alerts.yml

```diff
@@ -214,6 +214,17 @@ groups:
             will be full in less than 5 days assuming the average fill-up
             rate of the past 48 hours.
 
+      - alert: MTU Mismatch
+        expr: node_network_mtu_bytes{device!="lo"} != on() group_left() quantile(0.5, node_network_mtu_bytes{device!="lo"})
```

Comment:
I think this could be another approach:
```
topk(1, count_values("mtu", node_network_mtu_bytes) by(device)) by(device)
```
I tested this in one environment and I'm getting:
![image](https://user-images.githubusercontent.com/37327689/105081453-80f1fe00-5a92-11eb-91ba-77233423e851.png)

However I have no clue on how to compare a value with a label (mtu)...

Reply:
Thanks @epuertat . I have tried this approach before..but there is no way to compare the resultant metrics (mtu="1500") with the left side metrics (node_network_mtu_bytes)..we can only compare the value (3)..so to make this work we need to have the `mtu=1500` as a value (not in the element side).